kafka-ui-react-app/src/components/contexts/TopicMessagesContext.ts

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { SeekDirection } from 'generated-sources';
+
+export interface ContextProps {
+  seekDirection: SeekDirection;
+  searchParams: URLSearchParams;
+  toggleSeekDirection(val: string): void;
```

Comment:
`toggle` usually implies only two possible states **on** || **off** (true || false);

Let's try to fine more suitable name for this method. Eg: `setSeekDirection`

Reply:
okay will do.